.gitattributes

```diff
@@ -0,0 +1 @@
+licenses.html linguist-vendored
```

Comment:
Should I do?:

```suggestion
/licenses.html linguist-vendored
```

Reply:
I've decided it doesn't matter and that any `licenses.html` in this repo need not count toward our stats.